src/active_tile_data.cpp

```diff
@@ -376,6 +377,47 @@ void vehicle_connector_tile::load( JsonObject &jo )
     jo.read( "connected_vehicles", connected_vehicles );
 }
 
+void countdown_tile::update_internal( time_point to, const tripoint_abs_ms &p,
+                                      distribution_grid &grid )
```

Comment:
```suggestion
                                      distribution_grid & )
```
This will get rid of the `parameter 'grid' is unused` warnings